resources/views/users/manage.blade.php

```diff
@@ -0,0 +1,97 @@
+@extends('layouts.app')
+
+@include('includes.datatables')
+
+@section('title', trans('nav.settings.users'))
+
+@section('content-header')
+    <h1>
+        {{ trans('nav.settings.users') }}
+        <small></small>
+    </h1>
+    <ol class="breadcrumb">
+        <li><a href="{{ url('/') }}"><i class="fa fa-home"></i> {{ trans('nav.home') }}</a></li>
+        <li class="active">{{ trans('nav.settings.users') }}</li>
+    </ol>
+@endsection
+
+@section('content')
+    <div style="margin-bottom: 15px">
+    <button type="button" class="btn btn-primary showModal" data-href="{{ route('users.create') }}" data-toggle="modal"
+            data-target="#generalModal" data-modal-title="{{ trans('user.manage.create') }}">
+        <i class="fa fa-user-plus"></i> {{ trans('user.manage.create') }}
+    </button>
+    </div>
+
+    {!! $dataTable->table(['class' => 'table table-hover']) !!}
+
+    <!-- Modals -->
+    @include('components/modals/general')
+    @include('components/modals/delete', trans('user.manage.deleteconfirm'))
+@endsection
+
+@section('scripts')
+    {!! $dataTable->scripts() !!}
+    @include('includes.modal')
+    <script type="text/javascript">
+        $(document).on('click', '.saveUser', function(e){
+            e.preventDefault();
+
+            var $this = $(this);
+            var form = $('.userForm');
+            var table = window.LaravelDataTables["dataTableBuilder"];
+
+            $.ajax({
+                url: 	form.attr('action'),
+                type: 	form.attr('method'),
+                data: 	form.serialize(),
+                cache: 	false,
+                dataType: 'json',
+                success:function(data) {
+                    table.ajax.reload();
```

Comment:
I did a composer update so whichever is the latest one.

Do you need window. or can you just drop that?  Or maybe even just utilise LaravelDataTables['dataTableBuilder'].draw(false); direct rather than assigning it to a variable?